src/Symfony/Component/ErrorRenderer/Command/DebugCommand.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\ErrorRenderer\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Exception\InvalidArgumentException;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\ErrorRenderer\ErrorRenderer\ErrorRendererInterface;
+use Symfony\Component\ErrorRenderer\Exception\FlattenException;
+
+/**
+ * A console command for retrieving information about error renderers.
+ *
+ * @author Yonel Ceruto <yonelceruto@gmail.com>
+ */
+class DebugCommand extends Command
+{
+    protected static $defaultName = 'debug:error-renderer';
+
+    private $renderers;
+
+    /**
+     * @param ErrorRendererInterface[] $renderers
+     */
+    public function __construct(array $renderers)
+    {
+        $this->renderers = $renderers;
+
+        parent::__construct();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure(): void
+    {
+        $this
+            ->addArgument('format', InputArgument::OPTIONAL, sprintf('Outputs a sample in a specific format (one of %s)', implode(', ', array_keys($this->renderers))))
```

Comment:
Should be an option to be consistent with other commands.

Reply:
Here "format" doesn't have the same meaning as in other commands. Other commands use the `---format` option to change the current output to another format, here the format shows a different output with a different meaning "Outputs an error sample in a specific format".

It could also be confusing because people would expect that with `---format=json` the list of available renderers in the specified format and that is not the case.

Do you think the description of the argument should be improved to make it clearer?

Reply:
It is rather "I want to see an output of the error renderer associated with this format".

Reply:
@fabpot is this thread still a blocker?

Reply:
I think having this as argument makes sense in this case.